data/data/ibmcloud/master/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "ip_addresses" {
```

Comment:
This should be named `control_plane_ips` to be picked up by default by the bootstrap gather.
```suggestion
output "control_plane_ips" {
```

Reply:
okay, will update